records_mover/records/schema/field/sqlalchemy.py

```diff
@@ -47,8 +47,7 @@ def field_from_sqlalchemy_column(column: Column,
                                  type_=type_,
                                  driver=driver)
     elif (isinstance(type_, (sqlalchemy.sql.sqltypes.Numeric,
```

Comment:
Hmm... Is this intentional?

Reply:
Yep!  It turns out the postgres DOUBLE_PRECISION entry here wasn't needed, as its parent class is `sqlalchemy.sql.sqltypes.Float`